pkg/exporter/ipfix_proto.go

```diff
@@ -0,0 +1,498 @@
+package exporter
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+
+	"github.com/netobserv/netobserv-ebpf-agent/pkg/flow"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware/go-ipfix/pkg/entities"
+	ipfixExporter "github.com/vmware/go-ipfix/pkg/exporter"
+	"github.com/vmware/go-ipfix/pkg/registry"
+)
+
+var ilog = logrus.WithField("component", "exporter/IPFIXProto")
+
+type IPFIXProto struct {
```

Comment:
I'd name this struct as IPFIX (and the file as `ipfix.go` to avoid confusion).

The `GRPCProto` name was derived from `GRPC+Protocol Buffers`. Since this doesn't use protocol buffers, you can remove the `Proto` prefix.

Reply:
Sure, Will change it.